Sources/KSCrashRecording/include/KSCrashConfiguration.h

```diff
@@ -152,6 +152,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, assign) BOOL enableSwapCxaThrow;
 
+/** If true, enables monitoring for SIGTERM signals.
+ *
+ * A SIGTERM is usually sent to the application by the OS during a graceful shutdown,
+ * but it can also happen on some Watchdog events.
+ * Enabling this can provide more insights into the cause of the SIGTERM, but
+ * it can also generate many false-positive crash reports.
+ *
+ * **Default**: false
+ */
+@property(nonatomic, assign) BOOL enableSigTermMonitoring;
```

Comment:
When I saw this issue, I thought it might be a good idea to let users modify `g_fatalSignals` with a config. 
https://github.com/kstenerud/KSCrash/blob/00ca0b03976877edad04e1868662d6f07943b879/Sources/KSCrashRecordingCore/KSSignalInfo.c#L99-L101
What do you think about this?

Reply:
It sounds fine, but do you have any other example where we would want to exclude some other signal from this list?

Reply:
Ok, I was not thinking of disabling, but of adding other arbitrary signals. But now I understand that this is out of the scope of the library and may complicate the API.